usagov/usagov-2021
# USAGOV-1594-tome-basepath: On export, Check basepath to exclude configured paths

When exporting to a subdirectory, account for the base path when we test if a directory should be excluded from export.


## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1594

## Description
When testing if a path should be exported or excluded, adds a check to see if the path starts with the basepath when tome is exporting into a subdirectory of the document root.



## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


Export the site with a URI like:

```
drush tome:static --process-count=80 --uri="https://static.example.com/static/export/"
```

Your export folder should not have files from directories in `$settings['usagov_tome_static_path_exclude_directories']`. You should not see a `html/static/export/node` directory, `html/static/export/node` directory, `html/static/export/es/node` directory, `html/static/export/jsonapi` directory, etc.

### Requires New Config

### Requires New Content

### Validation Steps
- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
